resources/views/inputs/includes/input-label.blade.php

```diff
@@ -23,6 +26,12 @@ class="items-center input-label @error($name) input-label--error @enderror"
     @endif
 
     @if ($tooltip)
-        <div class="input-tooltip" data-tippy-content="{{ $tooltip }}">?</div>
+        <div class="{{ $tooltipClass }}" data-tippy-content="{{ $tooltip }}">
+            @if($tooltipType === 'info')
+                <x-ark-icon name="hint" :size="$tooltipIconSize" />
```

Comment:
I meant to say `x-ark-info` like you used before, not icon. This only adds duplication :sadup: